### PR TITLE
Revert incorrect guidance on TeamsJS version usage

### LINKED ISSUE #10406: TeamsJS version for Teams-only apps
Which version of the Teams JavaScript client library should we use in our Teams-only Angular app? Our app appears in Teams Tabs as an iFrame.

This is confusing in your documentation because the best practices differ from the recommended version and it is unclear which version we should use.
Related part of the documentation:  [Teams-only apps](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-library?tabs=javascript%2Cmanifest-teams-toolkit#teams-only-apps)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4ae8187e-a7e4-4b5a-6f38-da6bfa30f2d2
* Version Independent ID: 4ae8187e-a7e4-4b5a-6f38-da6bfa30f2d2
* Content: [Teams JavaScript client library SDK - Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-library?tabs=javascript%2Cmanifest-teams-toolkit#teams-only-apps)
* Content Source: [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/tabs/how-to/using-teams-client-library.md)
* Service: **msteams**
* Sub-service: **m365apps**
* GitHub Login: @erikadoyle
* Microsoft Alias: **mosdevdocs**

### PR DESCRIPTION
**Pull request by erikadoyle**
This reverts commit a761d3c67c97bd1107d7a135830b5cea602f8d65.

Fixes #10406 

Also includes a "hotfix" for screenshot I recently updated incorrectly (showing duplicate M365 app UI, instead of M365 + Outlook)

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5d5653baf7a391e5423eb12996deaea611fb2e7b/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 87 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ef873ff9-ac34-4850-904e-98324e0f8ef3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine.

File: msteams-platform/tabs/how-to/using-teams-client-library.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ An API translation layer (mapping v.1 to v.2 TeamsJS API calls) is provided to e
 
 #### Teams-only apps
 
-Even if you intend your app to only run in Teams (and not Microsoft 365 app and Outlook), best practice is to start referencing the latest TeamsJS (*v.2.0* or later) as soon as convenient, in order to benefit from the latest improvements, new features, and support (even for Teams-only apps). We recommend you to use TeamsJS v.1.13.1 as it's supported, but no new features or improvements are added.
+Even if you intend your app to only run in Teams (and not Microsoft 365 app and Outlook), best practice is to start referencing the latest TeamsJS (*v.2.0* or later) as soon as convenient, in order to benefit from the latest improvements, new features, and support (even for Teams-only apps). TeamsJS v.1.12 will continue to be supported, but no new features or improvements will be added.
```
Comment (v-ypalikila) [inline msteams-platform/tabs/how-to/using-teams-client-library.md:51]: Hi @erikadoyle ,

Can we remove the part "but no new features or improvements will be added" as it looks like a negative information for the user and would not change the sentence if removed.

Comment (erikadoyle) [reply msteams-platform/tabs/how-to/using-teams-client-library.md]: Done.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/149d43e9a8dc0f2b74bcb503c165a604b666c319/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62dda64e-422a-4c16-8883-2c7376f0d370) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9f640ac5cb21b421aeadf7de10f9de0ebd67b81c/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4a22e51f-ca55-4424-905f-de66214d3ca7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5bd2d511b24247031fda4011dd9dcfc737cd482d/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fa0a615b-fb1b-4d34-8d21-aef1732d55a2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/618e8fda25ef4ab502184a3706bbfb73d6664550/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 88 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d7dc8a58-7a05-4087-ad4c-76a4e366f532) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**